### ISSUE TITLE
[THIRDPARTY] SIRIUS Java issue

### ISSUE THREAD

**Issue by oliveralka** [OpenMS/OpenMS#3717]
Unfortunately, it seems that SIRIUS is not working with Java versions greater than 9.
This is due to an identifier in the code, which is not supported anymore. 

Associated Error:
```
Error:
Exception in thread "main" java.lang.ExceptionInInitializerError
        at de.unijena.bioinf.ms.cli.SiriusCLIApplication.main(SiriusCLIApplication.java:13)
Caused by: java.lang.NullPointerException
        at de.unijena.bioinf.sirius.core.ApplicationCore.<clinit>(ApplicationCore.java:104)
        ... 1 more
```

I am in contact with the SIRIUS developers.

**Comment by timosachsenberg** [comment]
I'll add to this issue:
on windows 10 (executed from KNIME), if no java has been installed previously, the java homepage pops up. Installation of java and restart of KNIME resolves the issue.

**Comment by jpfeuffer** [comment]
As a note, I would prefer that Sirius uses the KNIME internal Java unless otherwise specified. If we have it at hands, why not save the user some hassle (assuming that the versions are compatible).

**Comment by oliveralka** [comment]
The first issue will be fixed with next Sirius release.

**Comment by oliveralka** [comment]
Will be fixed with the update to SIRIUS 4.0.1.
